test/functional/Jsr292/src_110/com/ibm/j9/jsr292/MethodHandleTestExtra.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
```

Comment:
If this contains tests only applicable to JDK11+, can the name of the file reflect that?